docs/core/howto/defer-intro.rst

```diff
@@ -323,6 +323,7 @@ Python 3.5 introduced :pep:`492` ("Coroutines with async and await syntax") and
 :py:meth:`Deferred.fromCoroutine <twisted.internet.defer.Deferred.fromCoroutine>` allows you to write coroutines with the ``async def`` syntax and ``await`` on Deferreds, similar to ``inlineCallbacks``.
 Rather than decorating every function that may ``await`` a Deferred (as you would with functions that ``yield`` Deferreds with ``inlineCallbacks``), you only need to call ``fromCoroutine`` with the outer-most coroutine object to schedule it for execution.
 Coroutines can ``await`` other coroutines once running without needing to use this function themselves.
+Coroutines can be passed to ``yield`` in code based on :py:func:`inlineCallbacks <twisted.internet.defer.inlineCallbacks>`.
```

Comment:
Maybe add this ... not to hint that this is from 16.4

But I think that it's best to just delete the `..note` from line 318

```suggestion

.. versionadded:: Twisted NEXT

Coroutines can be passed to ``yield`` in code based on :py:func:`inlineCallbacks <twisted.internet.defer.inlineCallbacks>`.
```

Reply:
Thanks, applied both.